### DISCUSSION TITLE
WGDashboard update failed

### DISCUSSION THREAD

**Discussion by Bogo268** [tteck/Proxmox#3550] [Help Needed] [answered]
Hello,
In the Wireguard LXC I can’t update the WGdashboard, when using “update” command and selecting option 2.
Always returns “update canceled”
Thank you for your work.

**Comment by kiwinh0** [comment]
This is due to the python version. Since the script is based on Debian 11, the version of python available in the stable channel is 3.9 and for WGDashboard 4.0, at least 3.10 is required. ;)

**Reply by Bogo268** [reply]
Thank you !

**Comment by andreportilho** [comment]
Leaving the manual installation for python 3.10 on debian 11 from wgdashboard documentation. https://github.com/donaldzou/WGDashboard#debian-1110

Run only what comes before git clone and use "update" to install wgdashboard and follow "Autostart WGDashboard on boot" too

**Comment by kiwinh0** [comment]
The ideal would be to make the leap to 12. Let's wait for the fantastic team of @tteck  🤗

**Reply by tteck** [reply, marked as answer]
https://github.com/tteck/Proxmox/blob/main/CHANGELOG.md

Debian 12.6 with WGDashboard 4

**Reply by kiwinh0** [reply]
I am missing this to be able to access the internet from the container 

`iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE`

to do it permanently:

```
apt-get install iptables-persistent
netfilter-persistent save
netfilter-persistent reload
```

Thank you for the incredible work you do

**Reply by ianalexis** [reply]
I was fighting with the internet connection until i tried this.
Thanks!
I think this must be added into the script.

**Reply by tteck** [reply]
Added
https://github.com/tteck/Proxmox/blob/253da053feff9c39142a58391eac08ccb6aa88f5/install/wireguard-install.sh#L25-L27

**Comment by ianalexis** [comment]
Please check this. IDK if it apply to this script but... [URGENT! Docker WGDashboard Container Security Vulnerability!](https://github.com/donaldzou/WGDashboard/issues/333)

**Reply by tteck** [reply]
Does not apply
